translations/es.json

```diff
@@ -329,8 +339,10 @@
   "rule.desc.CssTextHasContrast": " <p>El color del texto debe ser fácilmente visible y no debe ser el único indicador de significado o función. El balance de color debe tener al menos una proporción de 4.5:1 para texto pequeño y una proporción de 3:1 para texto grande. Advertencia: el uso de UDOIT para corregir una sección de texto puede invalidar el contraste en secciones de texto anidadas que no son del mismo color.</p>",
   "rule.desc.CssTextStyleEmphasize": "<p>Al enfatizar el texto, puede usar colores con suficiente contraste siempre que también aplique alguna otra forma de énfasis, como negrita o cursiva. Esto asegura que los usuarios de lectores de pantalla sean conscientes de la importancia del texto.</p>",
   "rule.desc.DocumentReadingDirection": "<p>Los cambios en la dirección del texto en el contenido en línea deben indicarse mediante cualquier elemento HTML (por ejemplo, <code>span</code>) con un atributo 'dir' que indique de izquierda a derecha o de derecha a izquierda. Por ejemplo, una frase hebrea dentro de un párrafo en inglés debe tener su propia dirección de texto indicada.</p>",
+  "rule.desc.EmbedTagDetected": "Es posible que el contenido incrustado no se pueda ver correctamente en dispositivos móviles. Tenga en cuenta a los usuarios móviles cuando se basen en objetos para contenido multimedia.",
   "rule.desc.FontIsNotUsed": "<p>La etiqueta <code>font</code> está obsoleta y no debe utilizarse. Investigar usando hojas de estilo en su lugar.</p>",
   "rule.desc.HeadersHaveText": "<p>Los usuarios videntes y <a href='https://es.wikipedia.org/wiki/Lector_de_pantalla' target='_blank'>lectores de pantalla</a> dependen de los encabezados para organizar el contenido de la página. Los títulos no deben estar vacíos y deben representar un esquema preciso del contenido.</p>",
+  "rule.desc.HeadingsInOrder": "Anida los encabezados por su rango (o nivel). El encabezado más importante tiene el rango 1 (h1) en los archivos HTML o el rango 2 (h2) en las páginas de Canvas, el encabezado menos importante tiene el rango 6 (h6). Los encabezados con un rango igual o superior inician una nueva sección, los encabezados con un rango inferior inician nuevas subsecciones que forman parte de la sección con mayor rango. Saltarse los rangos de encabezados puede ser confuso y debe evitarse.",
```

Comment:
"Organice" (organize) instead of "anida" fits better. I've personally never heard nesting used figuratively in Spanish.